meetupselector/user/tasks.py

```diff
@@ -1,6 +1,19 @@
 from celery import shared_task
+from django.conf import settings
+from django.utils.translation import gettext_lazy
+
+from meetupselector.mail import send_templated_email
 
 
 @shared_task(bind=True)
-def debug_task(self):
-    print(f"Request: {self.request!r}")
+def send_registration_mail(_, email: str, confirmation_url: str):
+    context = {
+        "project_name": settings.PROJECT_NAME,
+        "confirmation_url": confirmation_url,
```

Comment:
The `confirmation_url` is not a valid URL, it's only the endpoint. In the email it shows as: `/api/users/verify` so it can't be clicked.

How can do it as clickable URL?

Reply:
Good catch! Fixed :)